pkg/commands/build/build.go

```diff
@@ -21,23 +21,34 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/build"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/commands/cmdlib"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/files"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/filter"
 	log "github.com/golang/glog"
 	"github.com/spf13/cobra"
 )
 
 // options represents options flags for the build command.
-type options struct{}
+type options struct {
+	// annotations selects a subset of patch templates on which to apply options
+	// Has the form "foo,bar;biff,baz".
```

Comment:
I might go through and change all these to foo=bar,biff=baz. But I'll do that in a following PR.

Reply:
Filed as #163